### PR TITLE
BlackBeezMini 3d Board

### PR DESCRIPTION
**Pull request by I3DBeeTech**
### Description

The board is based on STM32F401CCU6 Blackpill.
It supports only Neopixel mini 12864 lcds, D6/D7 had no pins to share.
Single Extruder and hotbed model

### CODE REVIEW

File: Marlin/src/pins/stm32f4/pins_BLACKBEEZMINI.h [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+#define SD_MOSI_PIN                         PA7
+
+//LCD REPRAP_DISCOUNT_SMART_CONTROLLER
+//
+ #define LCD_PINS_RS                        PB12
+ #define LCD_PINS_EN                        PB13
+ #define LCD_PINS_D4                        PB14
+ #define LCD_PINS_D5                        PA3
+ //#define LCD_PINS_D6                      PB1   //
+ //#define LCD_PINS_D7                      PB2   ///<- Ты вонючий алкашь, что блядь это такое, сука, я тебя спрашиваю!
```
Comment (thisiskeithb) [inline Marlin/src/pins/stm32f4/pins_BLACKBEEZMINI.h]: That's quite the comment.

**Comment by thisiskeithb** [comment]
I pushed 6290863 to de-tab a couple files and I have a question about that comment left in the pins file since it's so vulgar...

File: Marlin/src/pins/stm32f4/pins_BLACKBEEZMINI.h (hunk 1 above)
Comment (EvilGremlin) [reply Marlin/src/pins/stm32f4/pins_BLACKBEEZMINI.h]: Да ещё и с грамматической ошибкой.

Comment (I3DBeeTech) [reply Marlin/src/pins/stm32f4/pins_BLACKBEEZMINI.h]: We failed to uncheck this line. took some old reference. we should have rechecked it.
Request to delete both lines which aren't used for any further 
//#define LCD_PINS_D6                      PB1   //
 //#define LCD_PINS_D7                      PB2   //

**Comment by thinkyhead** [comment]
> supports only Neopixel mini 12864 lcds

What kind of LCD connector does this board have? Is it the standard EXP1/2, single EXP connector, or something else?

**Comment by ellensp** [comment]
I request that you attach links to or images of the motherboard in question, pins and schematic diagrams.

**Comment by I3DBeeTech** [comment]
> > supports only Neopixel mini 12864 lcds
> 
> What kind of LCD connector does this board have? Is it the standard EXP1/2, single EXP connector, or something else?

Its the same connector used in regular boards 
Please do follow by the images below connected to 12864 lcd
(board in image is a testing board which is now had small updates and on production process)
The Schematics and all required Docs will be upload by this end of the month to our git, 

board1
board2
newboard

**Comment by thinkyhead** [comment]
> Its the same connector used in regular boards

In that case the pins file needs to define the EXP1 and EXP2 header pins like the other boards do.

**Comment by thinkyhead** [comment]
I went ahead and added my best guesses for EXP1/2 pins. Please correct my work, and add any others that were missed.

**Comment by I3DBeeTech** [comment]
> I went ahead and added my best guesses for EXP1/2 pins. Please correct my work, and add any others that were missed.

Heart-full Appreciations for the good support and Efforts.
The code looks clean and clear. Tested 
Thank you

**Comment by sjasonsmith** [comment]
When is this board going to be available for purchase? I realize it's probably something you have under development, but the only results Google provides are for this pull request.

I hesitate to want to merge new board support if they aren't available to the public. In the past we've had trouble understanding the risk of some changes, only to find out the "boards" impacted never really existed, and were added prematurely.

**Comment by I3DBeeTech** [comment]
> When is this board going to be available for purchase? I realize it's probably something you have under development, but the only results Google provides are for this pull request.
> 
> I hesitate to want to merge new board support if they aren't available to the public. In the past we've had trouble understanding the risk of some changes, only to find out the "boards" impacted never really existed, and were added prematurely.

We had completed the R&D part for this models and in production with export quality, Bee9 and this model will be available to market between December 19th - January 20th. To get into market we planned to make low cost as well reasonable margin product to attract Consumers and also our clients for their printers/Purpose.
Our website will relaunched with only our products. The present weblink is just showcase till december 19th - january 20th.
Also seriously waiting for the marlin release with our board which boost up our market is also a reason to hold. 

I request you understand that this isn't a stunt models. Docs, Datasheet, Schematics, Manuals will be updated on the same day of release.
Thank you for the time and efforts

**Comment by I3DBeeTech** [comment]
For reference :
Our company details : https://www.insiderbiz.in/company/I3DBEE-TECHNOLOGIES-PRIVATE-LIMITED
Our Website : https://i3dbee.com/ (temporary templet)

hi @thisiskeithb any update ?
Thank you

**Pull request merged**